### DISCUSSION TITLE
Shelly mini to new "homekit"

### DISCUSSION THREAD

**Discussion by jknutz** [RavenSystem/esp-homekit-devices#2276] [Q&A] [answered]
HI, i've syccesfully flashed and used my shelly mini 1pm in homekit, i want to move to a new "home" but can't move since it is registered to the old home even though i have deleted it in the homekit app. What to do? cant connect it to any home now.
hope to find some help

**Comment by RavenSystem** [comment]
You must enable setup mode, and check "Reset HomeKit ID" and "Remove Wifi settings".

**Comment by jknutz** [comment]
That’s what I thought. I just can’t connect to the web interface.

**Reply by RavenSystem** [reply]
You must enable setup mode first.
Is your device connected to your WiFi network?

**Reply by jknutz** [reply]
It is connected to my wifi, 192.168.4.36 is the ip it is getting.

**Reply by RavenSystem** [reply]
Next step:
Has device any physical switch or button connected?

**Reply by jknutz** [reply]
Yes it is connected to a physical switch and a light bulb

**Reply by RavenSystem** [reply, marked as answer]
Ok. Turn it on and off quickly 8 times to put device in setup mode. Then, access web using http://192.168.4.36:4567

**Reply by jknutz** [reply]
THX!!

**Reply by RavenSystem** [reply]
Now, move Shelly to the new location. Connect to WiFi AP `HAA-??????`, open http://192.168.4.1:4567 and configure your new WiFi settings. Then, add it to Apple Home App.

**Comment by jknutz** [comment]
It all works now thanks a millionMvh Jens-Henrik KuntzDen 19. jul. 2024 kl. 14.17 skrev José Antonio Jiménez Campos ***@***.***>:﻿
Now, move Shelly to the new location. Connect to WiFi AP HAA-??????, open http://192.168.4.1:4567 and configure your new WiFi settings. Then, add it to Apple Home App.

**Reply by RavenSystem** [reply]
FYI: There are several ways to put device in setup mode: https://github.com/RavenSystem/esp-homekit-devices/wiki/setup-mode#entering-setup-mode
